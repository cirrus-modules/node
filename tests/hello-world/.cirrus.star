load("../../lib.star", "hello_world")

def main(ctx):
    return [hello_world()]
