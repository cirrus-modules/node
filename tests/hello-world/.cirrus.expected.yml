task:
  name: Hello World
  container:
    image: debian:latest
    cpu: 2
    memory: 4096
  greeting_script:
    - echo "Hello, World!"
